### PR TITLE
Add GitHub Actions CI configuration

### LINKED ISSUE #6: Set up the CI
We should add a automated pipeline for testing

The pipeline should be validated before merging to develop and master

### PR DESCRIPTION
**Pull request by maximelouet**
Add a Build workflow to `docker build` and a Test workflow (unused for
now). They both execute on PRs and on the `develop` and `master`
branches.
Closes #6.

### CODE REVIEW

**Comment by maximelouet** [comment]
@AnastasiaPromtep we have no tests yet; should we comment the task in the workflow or add a basic Query test?

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  push:
+    branches:
+      - develop
+      - master
+
+jobs:
+
+  docker-build:
+
+    runs-on: ubuntu-latest
```
Comment (AnastasiaPromtep) [inline .github/workflows/build.yml:16]: Is there not an alpine something for this ?

Comment (maximelouet) [reply .github/workflows/build.yml:16]: No, there's very little choice (https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on).
But the jobs seems to execute very quickly so I don't mind :slightly_smiling_face:

Comment (AnastasiaPromtep) [reply .github/workflows/build.yml:16]: > @AnastasiaPromtep we have no tests yet; should we comment the task in the workflow or add a basic Query test?

I think it is best to comment it for the moment

**Pull request merged**